Assets/Scripts/Game/GamePack.cs

```diff
@@ -10,14 +10,19 @@ public class GamePack
 	private string root;
 
 	/// <summary>
-	/// The desired ISO 639-2 language code.
+	/// The default ISO 639-2 language code.
```

Comment:
Change this back to desired. It doesn't make sense to have a default that changes during runtime.